homeassistant/components/linky/sensor.py

```diff
@@ -145,8 +145,8 @@ def device_state_attributes(self):
     def device_info(self):
         """Return device information."""
         return {
-            "identifiers": {(DOMAIN, self.unique_id)},
-            "name": self.name,
+            "identifiers": {(DOMAIN, "meter")},
```

Comment:
Is there only one meter possible for the whole integration?

Reply:
Yes, one username/password correspond to one meter.

Reply:
We have to connect the entity device info to the correct device. Since more than one device or account is possible per this integration, we can't just use `DOMAIN` and `"meter"` as I'd of the device. The config entry id could work.

Reply:
Maybe we can do this to allow multiple linky integrations ?
```python
{ 
  "identifiers": {(DOMAIN, self.username)})
}
```

Reply:
Ok.